bin/auth.js

```diff
@@ -11,23 +11,28 @@ let winston = require('winston');
 let jwt = require('jsonwebtoken');
 let fs = require('fs');
 let _ = require('underscore');
+const mongoose = require('mongoose');
 
 let config = require('../api/config');
+delete config.event;  // Hack to not connect to rabbitmq @TODO rework this
```

Comment:
Can you expand on this? Why do we need to do this?

Reply:
Right, so right now the code under config will always connect to rabbitmq once importing `common`, except when there's no config for it (`config.event`). Given the config will have the event configured, since it is shared with the main API, this CLI tool will not exit once it did its task. So I deleted the `event` key so it did not connect to rabbit.